### PR TITLE
add install script

### PR DESCRIPTION
**Pull request by trddddd**
#81

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		;;
+	esac
+}
+
+install-requirements() {
+	case $(get-distribution) in
+		ubuntu)
+			sudo apt-get update -y
+			udo apt-get install -y apt-transport-https \
+				git \
```
Comment (trddddd) [inline install.sh]: sudo

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+install-docker-compose() {
+	sudo curl -L "https://github.com/docker/compose/releases/download/$DOCKER_COMPOSE_VERSION/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+	sudo chmod +x /usr/local/bin/docker-compose
+	sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+}
+
+start-vpnbilling() {
+	git clone $SMARTBILLING_REPO
```
Comment (trddddd) [inline install.sh]: =(

**Comment by Mehonoshin** [comment]
@trddddd any progress?

**Comment by trddddd** [comment]
PR in https://github.com/Mehonoshin/smartvpn/pull/10/files

**Pull request closed without merging**